### PR TITLE
add note for filtered out studies in cross cancer histogram

### PR DESCRIPTION
**Pull request by JiaoJiao123**

### CODE REVIEW

**Comment by JiaoJiao123** [comment]
@jjgao The note for cross cancer histogram is added as we discussed. Please check it out when you get a chance. Thank you!

**Comment by JiaoJiao123** [comment]
Issue #994

**Comment by JiaoJiao123** [comment]
Issue #1032

File: portal/src/main/webapp/js/src/crosscancer.js [hunk 1]
```diff
@@ -123,30 +117,60 @@
...
             var histData = [];
             _.each(histDataOrg, function(study) {
                 cancerTypeCheck = true; 
-                var showStudy = $("#histogram-remove-study-" + study.studyId).is(":checked");
-                if(cancerTypes !== "all" && metaData.type_of_cancers[metaData.cancer_studies[study.studyId].type_of_cancer] !== cancerTypes){
-                    cancerTypeCheck = false;
-                }    
-     
-                if(!study.skipped && showStudy && cancerTypeCheck){
+                var showStudy = true;//$("#histogram-remove-study-" + study.studyId).is(":checked");
```
Comment (jjgao) [inline portal/src/main/webapp/js/src/crosscancer.js]: could you remove the code that you commented out?

**Pull request merged**